src/meshlab/glarea.cpp

```diff
@@ -2014,6 +2014,51 @@ bool GLArea::viewFromFile()
     return true;
 }
 
+bool GLArea::saveViewToFile()
+{
+    QFileDialog saveDiag(this, tr("Save View To File"), "./", tr("View file (*.xml)"));
```

Comment:
I suppose that this is working, but you could also do it with a simple call to

QFileDialog::getSaveFileName


Which is easier to configure, and don't need to check number of files equals 1, etc...

But for me, it is also OK this way you did. Thanks for your collaboration

Reply:
If I follow this suggestion, there is some logic below that needs to be modified, when it comes to the user not providing an .xml extension and having to fill it in for them. That requires access to the file dialog object, and with your suggestion it won't be accessible. There is probably work around that. The logic I put in I copied from a different place in the code, and I tested it, so it does the job. I am sure the whole thing can be improved, both here and in other places, but would require some time and careful testing to ensure all the scenarios are handled.